### PR TITLE
Added Linkerd guide

### PR DESCRIPTION
**Pull request by alparslanavci**

### CODE REVIEW

**Review by leszko — COMMENTED** [review]
Nice guide @alparslanavci 👏 

Added some comments in the https://github.com/hazelcast-guides/linkerd repo. Other than that, it looks good.

File: antora-playbook.yml [hunk 1]
```diff
@@ -66,6 +66,8 @@ content:
       start_path: doc
     - url: https://github.com/hazelcast-guides/ec2-cluster
       start_path: doc
+    - url: https://github.com/hazelcast-guides/linkerd
+      start_path: doc
```
[the remark this answers is not in the document]
Comment (alparslanavci) [reply antora-playbook.yml:70]: @leszko updated.

**Comment by alparslanavci** [comment]
@leszko thank you for the review, I updated the guide accordingly. Do we need anything else other than merging this PR?

**Review by leszko — APPROVED** [review]
LGTM 👍 

Thanks for all the changes, @alparslanavci

**Pull request merged**